### PR TITLE
[1.13.x-blue-next] [RHPAM-4560] - Adjust rhpam-kogito-builder-rhel8 image to enable lice…

### PR DESCRIPTION
**Pull request by spolti**
…nse compliance via publication of source containers

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by radtriste — APPROVED** [review]

**Comment by spolti** [comment]
Jenkins run PROD

**Comment by spolti** [comment]
Jenkins run prod

**Comment by spolti** [comment]
Jenkins RHPAM test

**Comment by radtriste** [comment]
@spolti We should merge https://github.com/kiegroup/kogito-images/pull/1521 and its backport to 1.13.x-blue if we want the Jenkins PR checks on the next branch

**Review by rgdoliveira — APPROVED** [review]

**Comment by radtriste** [comment]
jenkins run prod

**Comment by radtriste** [comment]
@spolti enjoy the checks :)

**Comment by kie-ci1** [comment]
**PR job** `#1` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/1/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare offline kogito-examples)<br/>Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/1.13.x-blue/pullrequest/kogito-images.prod.next/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Wed May 10 05:56:58 EDT 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Wed May 10 05:56:58 EDT 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/1/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci1** [comment]
**PR job** `#1` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/1/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Prepare offline kogito-examples)<br/>Stage "Prepare offline kogito-examples" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Build & Test Images)<br/>Stage "Build & Test Images" skipped due to earlier failure(s)<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/workspace/KIE/kogito/1.13.x-blue/pullrequest/kogito-images.next/workspaces<br/>[Pipeline] sh<br/>++ docker ps -a -q<br/>+ docker rm -f<br/>"docker rm" requires at least 1 argument.<br/>See 'docker rm --help'.<br/><br/>Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]<br/><br/>Remove one or more containers<br/>+ date<br/>Wed May 10 05:57:12 EDT 2023<br/>[Pipeline] sh<br/>++ docker images -q<br/>+ docker rmi -f<br/>"docker rmi" requires at least 1 argument.<br/>See 'docker rmi --help'.<br/><br/>Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]<br/><br/>Remove one or more images<br/>+ date<br/>Wed May 10 05:57:13 EDT 2023<br/>[Pipeline] sh<br/>+ rm -rf /home/jenkins/.cekit/cache<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/1/api/json<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by radtriste** [comment]
ok rebase is needed ^^

**Comment by spolti** [comment]
Rebased, thanks @radtriste!

**Comment by spolti** [comment]
jenkins run prod

**Comment by kie-ci1** [comment]
**PR job** `#2` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/2/display/redirect

**Test results:**
- PASSED: 38
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/2/testReport/TESTS-features.image.bamoe-kogito-builder-jvm/xml/[failed_to_read]/">TESTS-features.image.bamoe-kogito-builder-jvm.xml.[failed-to-read]</a></summary>
Failed to read test report file /home/jenkins/workspace/KIE/kogito/1.13.x-blue/pullrequest/kogito-images.prod.next/workspaces/bamoe-kogito-builder-rhel8/target/test/results/TESTS-features.image.bamoe-kogito-builder-jvm.xml<br/>org.dom4j.DocumentException: Error on line 134 of document  : An invalid XML character (Unicode: 0x1b) was found in the CDATA section.<br/>	at org.dom4j.io.SAXReader.read(SAXReader.java:511)<br/>	at org.dom4j.io.SAXReader.read(SAXReader.java:392)<br/>	at hudson.tasks.junit.SuiteResult.parse(SuiteResult.java:177)<br/>	at hudson.tasks.junit.TestResult.parse(TestResult.java:384)<br/>	at hudson.tasks.junit.TestResult.parsePossiblyEmpty(TestResult.java:314)<br/>	at hudson.tasks.junit.TestResult.parse(TestResult.java:256)<br/>	at hudson.tasks.junit.TestResult.parse(TestResult.java:242)<br/>	at hudson.tasks.junit.TestResult.parse(TestResult.java:220)<br/>	at hudson.tasks.junit.TestResult.<init>(TestResult.java:174)<br/>	at hudson.tasks.junit.JUnitParser$ParseResultCallable.invoke(JUnitParser.java:176)<br/>	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3578)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:211)<br/>	at hudson.remoting.UserRequest.perform(UserRequest.java:54)<br/>	at hudson.remoting.Request$2.run(Request.java:377)<br/>	at hudson.remoting.InterceptingExecutorService.lambda$wrap$0(InterceptingExecutorService.java:78)<br/>	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)<br/>	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)<br/>	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)<br/>	at java.base/java.lang.Thread.run(Thread.java:829)<br/>Caused by: org.xml.sax.SAXParseException; lineNumber: 134; columnNumber: 55; An invalid XML character (Unicode: 0x1b) was found in the CDATA section.<br/>	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)<br/>	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1465)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanCDATASection(XMLDocumentFragmentScannerImpl.java:1631)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2932)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)<br/>	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)<br/>	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)<br/>	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)<br/>	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)<br/>	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)<br/>	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)<br/>	at org.dom4j.io.SAXReader.read(SAXReader.java:494)<br/>	... 18 more<br/>
</details>

**Comment by radtriste** [comment]
issue: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/2/artifact/target/test/results/TESTS-features.image.bamoe-kogito-builder-jvm.xml

**Comment by kie-ci1** [comment]
**PR job** `#2` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/2/display/redirect

**Test results:**
- PASSED: 53
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/2/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

File: tests/features/bamoe-kogito-builder-jvm.feature [hunk 1]
```diff
@@ -31,18 +31,18 @@ Feature: rhpam-kogito-builder-rhel8 feature.
...
-      | RUNTIME_TYPE   | quarkus        |
-      | NATIVE         | true           |
-      | KOGITO_VERSION | 1.13.1-SNAPSHOT |
-    # Then s2i build log should contain does not supports native builds # looks like it does not work ...
+    Given s2i build /tmp/kogito-examples from dmn-example using nightly-1.13.x-blue and runtime-image ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:latest
+      | variable       | value           |
+      | RUNTIME_TYPE   | quarkus         |
+      | NATIVE         | true            |
+      | KOGITO_VERSION | 1.13.2-SNAPSHOT |
+    Then s2i build log should contain Container Image ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8 does not supports native builds, please refer to the documentation.
```
Comment (radtriste) [inline tests/features/bamoe-kogito-builder-jvm.feature]:
```suggestion
    # Then s2i build log should contain does not supports native builds # looks like it does not work ...
```

File: tests/features/bamoe-kogito-builder-jvm.feature [hunk 2]
```diff
@@ -31,18 +31,18 @@ Feature: rhpam-kogito-builder-rhel8 feature.
     And file /home/kogito/.m2/settings.xml should contain <url>https://maven.repository.redhat.com/techpreview/all</url>
 
   Scenario: Check if the expected message is printed if native build is enabled
-    Given failing s2i build /tmp/kogito-examples from dmn-example using 1.13.x
-      | variable       | value          |
-      | RUNTIME_TYPE   | quarkus        |
-      | NATIVE         | true           |
-      | KOGITO_VERSION | 1.13.1-SNAPSHOT |
+    Given s2i build /tmp/kogito-examples from dmn-example using nightly-1.13.x-blue and runtime-image ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:latest
```
Comment (radtriste) [inline tests/features/bamoe-kogito-builder-jvm.feature]:
```suggestion
    Given failing s2i build /tmp/kogito-examples from dmn-example using nightly-1.13.x-blue
```

**Comment by radtriste** [comment]
jenkins run prod

**Comment by kie-ci1** [comment]
**PR job** `#3` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/3/display/redirect

**Test results:**
- PASSED: 44
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.prod.next/3/testReport/features.image.bamoe-kogito-builder-jvm.rhpam-kogito-builder-rhel8 feature//Verify_that_the_Kogito_Maven_archetype_is_generating_the_project_and_compiling_it_correctly/">features.image.bamoe-kogito-builder-jvm.rhpam-kogito-builder-rhel8 feature..Verify that the Kogito Maven archetype is generating the project and compiling it correctly</a></summary>
Phrase 'Using Quarkus com.redhat.quarkus.platform:quarkus-maven-plugin:' was not found in the output of S2I
</details>

**Comment by radtriste** [comment]
archetype test failure is expected

**Comment by kie-ci1** [comment]
**PR job** `#3` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/3/display/redirect

**Test results:**
- PASSED: 53
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/3/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

**Comment by kie-ci1** [comment]
**PR job** `#4` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/4/display/redirect

**Test results:**
- PASSED: 53
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images.next/4/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

**Pull request merged**